.circleci/config.yml

```diff
@@ -48,6 +48,16 @@ jobs:
       - run: ./ci/install_bazelisk.sh
       - run: ./ci/do_ci.sh bazel.asan
 
+  format:
+    resource_class: xlarge
+    docker:
+      - image: ubuntu:18.04
+    steps:
+      - checkout
+      - run: ./ci/setup_ci_environment.sh
+      - run: ./ci/install_format_tools.sh
+      - run: ./ci/do_ci.sh format
```

Comment:
`format`'s missing from the workflows' jobs

Reply:
Oops! Thanks!